bin/migrate-oats-data/applications/submissions/review/lg_submission_review.py

```diff
@@ -0,0 +1,191 @@
+    )
+
+
+def _insert_records(conn, cursor, rows):
+    number_of_rows_to_insert = len(rows)
+
+    if number_of_rows_to_insert > 0:
+        insert_query = _compile_review_insert_query(number_of_rows_to_insert)
+        rows_to_insert = _prepare_data_to_insert(rows)
+        cursor.execute(insert_query, rows_to_insert)
+        conn.commit()
+
+
+def _compile_review_insert_query(number_of_rows_to_insert):
+    reviews_to_insert = ",".join(["%s"] * number_of_rows_to_insert)
+    return f"""
+                        INSERT INTO alcs.application_submission_review(
+                            first_name, 
+                            last_name,
+                            department, 
+                            position, 
+                            email, 
+                            phone_number, 
+                            is_ocp_designation, 
+                            ocp_bylaw_name,
+                            ocp_designation,
+                            ocp_consistent,
+                            is_subject_to_zoning,
+                            zoning_bylaw_name,
+                            zoning_designation,
+                            zoning_minimum_lot_size,
+                            is_zoning_consistent,
+                            application_file_number,
+                            local_government_file_number,
+                            audit_created_by
+                        )
+                        VALUES{reviews_to_insert}
+                        ON CONFLICT DO NOTHING;
+    """
+
+
+def _prepare_data_to_insert(rows):
+    row_without_last_element = []
+    for row in rows:
+        mapped_row = _map_data(row)
+        row_without_last_element.append(tuple(mapped_row.values()))
+
+    return row_without_last_element
+
+
+def _map_data(row):
+    return {
+        "first_name": row["first_name"],
+        "last_name": row["last_name"],
+        "department": row["contact_department"],
+        "position": row["title"],
+        "email": row["email_address"],
+        "phone_number": row["phone_number"],
+        "is_ocp_designation": _map_ocp_designation(row),
+        "ocp_bylaw_name": row["community_pln_bylaw_name"],
+        "ocp_designation": row["community_pln_designation"],
+        "ocp_consistent": _map_ocp_consistency(row),
+        "is_subject_to_zoning": _map_zoning(row),
+        "zoning_bylaw_name": row["zoning_bylaw_name"],
+        "zoning_designation": row["zoning_designation"],
+        "zoning_minimum_lot_size": row["minimum_lot_hectares"],
+        "is_zoning_consistent": _map_zoning_consistency(row),
+        "application_file_number": row["alr_application_id"],
+        "local_government_file_number": row["local_gov_file_number"],
+        "audit_created_by": OATS_ETL_USER,
+    }
+
+
+
+
+def _map_ocp_designation(ocp_value):
+
+    if ocp_value["community_pln_compliance_ind"] == 'X':
```

Comment:
Is this meant to be 'Y'?

Reply:
You would think that but no, that is the correct mapping. 
`if community_pln_designation = X -> false , if community_pln_designation = N -> true`

Reply:
Weird...